### DISCUSSION TITLE
License metadata on the website should not always show the latest version license

### DISCUSSION THREAD

**Discussion by MichaelTsengLZ** [npm/feedback#310] [Website] [answered]
When I chose an older version of a package, the license on npmjs.com always shows the license property from the latest package.json instead of from the version I picked.

I upload a test package [here](https://www.npmjs.com/package/@miczeng/test_npm_package/v/1.0.0), version 1.0.0 is MIT while 1.0.1 is Apache-2.0. The npmjs.com always shows the license of 1.0.1 even I choose 1.0.0. This will make people confused about the license in the past.

![image](https://user-images.githubusercontent.com/20408623/113179844-e2f28080-9204-11eb-8033-960575885b11.png)

**Comment by MylesBorins** [comment, marked as answer]
Thanks for raising this @MichaelTsengLZ 

We have an issue on our internal backlog tracking this. If I recall this was not as simple to fix as it seemed on surface, but I'll be prodding the team and looking at what is going on to see if there is a way we could get this fixed.

**Reply by lumaxis** [reply]
@MichaelTsengLZ I just deployed a change that should surface the version-specific license for a package on npmjs.com. I just took a look at your example package and that looks fine now to me. Would you mind giving it another look, too?
Thanks again for the suggestion!

**Reply by MichaelTsengLZ** [reply]
I will ask folks to take another look. Thank you so much for the fast change.
